### DISCUSSION TITLE
Addition of '/rules' command

### DISCUSSION THREAD

**Discussion by HackuJacku** [Zrips/CMI#6822] [Feature Requests]
Plugins such as EssentialsX or Nucleus (Sponge version of EssentialsX/CMI) all include a '/rules' command which when executed, displays a list of rules defined by the server administrators. 

In CMI, this feature is non-existent. Whilst adding a plugin to add this feature is possible, I believe that CMI should have this included. If implemented, the rules should be configurable and support Minecraft formatting.

An example of a solution is provided below:
![image](https://user-images.githubusercontent.com/18508413/163591836-b0d0767f-0dcc-46ce-896d-ee5d9fd579d9.png)

![image](https://user-images.githubusercontent.com/18508413/163591884-529efd9a-d44e-4b6a-af1a-ea2f6024d5da.png)

**Comment by lerndmina** [comment]
You can do this with CustomText and then making an alias to run the command to view the CustomText

**Comment by Zrips** [comment]
@HackuJacku 
As @lerndmina mentioned you can use /cmi editctext to create new custom text which can show defined text while performing command. Additionally there is simple option to enable alias command for it so that only ctext name is needed to be used as command for it to show up. On top of this you can use more complex text in it with auto or manual pagination. More information [https://www.zrips.net/cmi/custom-text/](https://www.zrips.net/cmi/custom-text/)

**Comment by mrfloris** [comment]
> Plugins such as EssentialsX or Nucleus (Sponge version of EssentialsX/CMI) all include a '/rules' command which when executed, displays a list of rules defined by the server administrators.
> 
> In CMI, this feature is non-existent. Whilst adding a plugin to add this feature is possible, I believe that CMI should have this included. If implemented, the rules should be configurable and support Minecraft formatting.
> 
> An example of a solution is provided below: ![image](https://user-images.githubusercontent.com/18508413/163591836-b0d0767f-0dcc-46ce-896d-ee5d9fd579d9.png)
> 
> ![image](https://user-images.githubusercontent.com/18508413/163591884-529efd9a-d44e-4b6a-af1a-ea2f6024d5da.png)

https://faq.cmi.support/rules
